exercises/01-composite-ui/README.md

```diff
@@ -125,3 +125,9 @@ Update the `orders` list template (`app\branding\orders\ordersView.html`) to dis
 <br />
 <strong>Order total:</strong> {{order.orderTotalPrice}}
 ```
+
+## Conclusion
+
+In this exercise we've seen how to combine data from various autonomous services into a user interface, in which the end-user is unaware of the architecture behind it.
```

Comment:
> in which the end-user is unaware of the architecture behind it

This doesn't feel right.  I'd leave it out, and perhaps call out that there is a _single_ user interface:

> In this exercise we've seen how to combine data from various autonomous services into a single user interface.